website/src/css/custom.css

```diff
@@ -41,24 +41,24 @@ Apply style on documentation
 /* You can override the default Infima variables here. */
 :root {
   --ifm-color-primary: #6d28d9;
-  --ifm-color-primary-dark: #7c3aed;
+  --ifm-color-primary-dark: #6223c5;
   --ifm-color-primary-darker: #5c21ba;
   --ifm-color-primary-darkest: #4c1b99;
   --ifm-color-primary-light: #7c3edd;
   --ifm-color-primary-lighter: #8349df;
-  --ifm-color-primary-lightest: #996ae5;
+  --ifm-color-primary-lightest: #8b54e1;
   --ifm-code-font-size: 95%;
 }
 
 /* For readability concerns, you should choose a lighter palette in dark mode. */
 [data-theme='dark'] {
-  --ifm-color-primary: #a778fa;
-  --ifm-color-primary-dark: #7c3aed;
-  --ifm-color-primary-darker: #5c21ba;
-  --ifm-color-primary-darkest: #4c1b99;
-  --ifm-color-primary-light: #7c3edd;
-  --ifm-color-primary-lighter: #8349df;
-  --ifm-color-primary-lightest: #996ae5;
+  --ifm-color-primary: #e5dbf5;
```

Comment:
Shouldn't the primary color be the same between light and dark? 
@mairin what do you think? I think you said you were also working on defining the color palettes for the tool itself, I believe we should use the same on the website.